js/app/articleHTMLRenderer.js

```diff
@@ -154,12 +154,22 @@ var ArticleHTMLRenderer = new Knowledge.Class({
             return '';
 
         function get_button_markup (network) {
-            let file = Gio.file_new_for_uri(`resource:///com/endlessm/knowledge/data/icons/scalable/apps/${network}-symbolic.svg`);
-            let [success, svg] = file.load_contents(null);
+            let file = Gio.file_new_for_path(`/usr/share/runtime/share/icons/EndlessOS/scalable/apps/${network}-symbolic.svg`);
+            let markup = null;
+
+	    try {
+                let [success, svg] = file.load_contents(null);
```

Comment:
nitpick: if you don't use the `success` variable, do `let [, svg] = file.load_contents(null)`